### DISCUSSION TITLE
Combined Select Popup + Slider  possible?

### DISCUSSION THREAD

**Discussion by lordhelmet77** [Clooos/Bubble-Card#999] [Questions about config, custom styles and templates]
![image](https://github.com/user-attachments/assets/23929590-251a-4a45-ab03-4bcd943cd37e)

Is something like this mockup possible, where I combined a slider and a select popup for the positions of a cover?

**Comment by MrBearPresident** [comment]
I don't think it is directly supported. (I'm not sure, I don't own any covers)
A way to do this is with a helper:
- Select helper

These you can make the desired button from (button-slider) with the cover as the entity. 
Add a sub-button that is the select helper. 

Afterwards make an automation that if the select-helper changes, you move the cover to the desired position. 


---

<sub>If this solves your issue, please mark this as solved.</sub>
